src/plugins/intel_cpu/src/nodes/kernels/kernel_utils.hpp

```diff
@@ -0,0 +1,547 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Because this file mostly contains `jitKernelBase`, I suggest to renamed it to jit_kernel_base.hpp

Reply:
Agree, will rename.

Reply:
Done